### ISSUE TITLE
Wrong layer indexed colors for dwg2004,2007,2010,2013,2018

### ISSUE THREAD

**Issue by zamtmn** [LibreDWG/libredwg#512]
layer indexed colors works only in dwg2000
[dwg_colors.zip](https://github.com/LibreDWG/libredwg/files/9229232/dwg_colors.zip)

**Comment by rurban** [comment]
I see properly read LAYER color.index, color.rgb and color.method for all layers.
e.g.
```
color.index: 256 [CMC.BS 62]
color.rgb: 0xc3000001 [CMC.BL 420]
color.method: 0xc3 Truecolor
color.flag: 0x0 [CMC.RC]
```

**Comment by rurban** [comment]
I've fixed downconverting the CMC index with 21a9f24ab9e914823aa1c6da7bb0980520d076c9
tag: 0.12.5.5852
